.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 
 node_js:
-  - 10
-  - 9
-  - 8
   - 6
-  - 4
+  - 8
+  - 9
+  - 10
```

Comment:
I flipped this order, because when Greenkeeper suggests changes to `.travis.yml` it appends the node version to the end of this list.